client/Fireport/Assets/Scripts/AnimationEngine/AnimationEngineBehaviour.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections.Generic;
+using Game;
+using Model.State;
+using UnityEngine;
+
+namespace AnimationEngine {
+    public class AnimationEngineBehaviour : MonoBehaviour {
```

Comment:
As long as one of these exists in your scene somewhere, you should be able to animate properly.  The crux of this engine is that the actual animations exist on the individual objects themselves.  This engine is only responsible for calling `Play` on them and watching for their `IsPlaying` to return false so that it knows the animation has completed.